examples/fine-tuning/create_dataset.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ] || ! [ -d "$2" ]; then
+  echo "Usage: $0 <NEW_DATASET_DIR> <MNIST_DIR>" >&2
+  exit 1
+fi
+
+NEW_DATASET_DIR=$1
+MNIST_DIR=$2
+
+# create new dataset directories
+mkdir -p $NEW_DATASET_DIR/odd
+mkdir -p $NEW_DATASET_DIR/even
+
+# create symbolic links
+for digit in 0 2 4 6 8
+   do cp $MNIST_DIR/$digit/*.png $NEW_DATASET_DIR/even/
```

Comment:
I actually used symlinks in my first implementation but noticed that shuffling didn't work across folders (?) so I was effectively only getting zeros for even numbers and ones for odd numbers... therefore I decided to do a hard copy in order to have a flat file structure. Does it make sense?